env.yml

```diff
@@ -0,0 +1,7 @@
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - pip
+  - python>=3.7,<3.11
```

Comment:
why not 3.11?

Reply:
This may be from a while back, something broke in datajoint-python with python 3.11 (diagram or connection?)
This may have already been resolved in latest `datajoint-python`

Reply:
It might be due to compatibility with the latest version of SpikeInterface. In fact, the latest release 0.101.1 has dropped support for Python<3.9.

Reply:
`datajont-python` should work for all `Python 3.9+`